gammapy/catalog/fermi.py

```diff
@@ -1686,6 +1884,56 @@ def __init__(self, filename="$GAMMAPY_DATA/catalogs/fermi/gll_psch_v13.fit.gz"):
         self.energy_bounds_table = Table.read(filename, hdu="EnergyBounds")
 
 
+class SourceCatalog2PC(SourceCatalog):
+    """Fermi-LAT 2nd pulsar catalog.
+
+    - https://ui.adsabs.harvard.edu/abs/2013ApJS..208...17A
+    - https://fermi.gsfc.nasa.gov/ssc/data/access/lat/2nd_PSR_catalog/
+
+    One source is represented by `~gammapy.catalog.SourceCatalogObject2PC`.
+
+    TODO  : Fix the UnitsWarning here ...
```

Comment:
could you make the TODO more explicit and move it within the code as comment, otherwise it will show in the doc.

Reply:
It is fixed, I removed it !